### DISCUSSION TITLE
why is the css style for the website called `bluedodger`?

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#99] [Q&A] [answered]
title

jus curious

**Comment by Rubiidev-18** [comment]
its called that because its the default site color, also, you should try making the dark theme on april 8th being eclipse orange, + change the colors in the theme

**Comment by mariocraft987** [comment, marked as answer]
and its the base color i used in the navbar

**Reply by rubiidev18alt** [reply]
I wish I can mark 2 answers

**Reply by mariocraft987** [reply]
eclipes orange???????????????????????

**Reply by rubiidev18alt** [reply]
that's the total eclipse theme, it's just dodgerblue rn, but you can edit the colors and make a theme switcher

**Reply by mariocraft987** [reply]
it doesn't work
